tests/unit/s2n_tls13_state_machine_handshake_test.c

```diff
@@ -394,14 +394,42 @@ int main(int argc, char **argv)
 
         conn->handshake.handshake_type = test_handshake_type;
 
-        for (int i=0; i < sizeof(expected) / sizeof(char *); i++) {
+        for (int i = 0; i < sizeof(expected) / sizeof(char *); i++) {
```

Comment:
Nit: I thought there was a macro for array sizes?

Reply:
ah yes, i changed it to s2n_array_len but switched back right before making this PR. will fix up